File: JavaScript-Fundamentals/Homeworks/Operators-and-Expressions/04. Third digit/ThirdDigit.js
function solve(args) {
    var number = args + '';
    var thirdDigit = 0;

    if (number.length >= 3) {
        thirdDigit = +number[number.length - 3];
    }

    return Math.floor(thirdDigit) === 7 ? true : 'false' + ' ' + thirdDigit;
}

File: Modul-I/01.C#PartOne/Homeworks/03.Operators-and-Expressions-Homework/ThirdDigit/ThirdDigit.cs
namespace ThirdDigit
{
    using System;

    class ThirdDigit
    {
        static void Main(string[] args)
        {
            int number = int.Parse(Console.ReadLine());
            int thirdDigit = 0;

            for (int i = 0; i < 3; i++)
            {
                if (i == 2)
                {
                    thirdDigit = number % 10;
                }
                else
                {
                    number /= 10;
                }

            }

            if (thirdDigit == 7)
            {
                Console.WriteLine("true");
            }
            else
            {
                Console.WriteLine("false {0}", thirdDigit);
            }

        }
    }
}
